com.playeveryware.eos/Runtime/Core/PlatformManager.cs

```diff
@@ -65,6 +66,7 @@ private struct PlatformInfo
             public Type ConfigType;
             public string DynamicLibraryExtension;
             public string PlatformIconLabel;
+            public Func<PlatformConfig> GetConfigFunc;
```

Comment:
This is not utilized, should be removed.

Reply:
Instead of removing it - it was left in with a slightly different implementation to support non-reflection solution. Marking conversation as resolved.